core/model/src/main/java/org/activityinfo/model/type/number/Quantity.java

```diff
@@ -9,11 +9,15 @@
 
 public class Quantity implements FieldValue {
 
+    @Deprecated
     public static final String UNKNOWN_UNITS = "unknown";
 
     private final double value;
+
+    @Deprecated
     private String units;
 
+    @Deprecated
     public Quantity(double value) {
```

Comment:
Why is this constructor deprecated?

Reply:
Ah, its to be replaced with a no-argument constructor

Reply:
double check. Seems like a Quantity(double) constructor would be useful. Did you mean to deprecate the constructor Quantity(double,String) below?

Reply:
Possibly, I'll have a look at it this morning and push the changes from this review.